export function catchErrors(err) {
    console.error(`Ошибка: ${err.status}`);
}